src/optimizer/OptimizerUtils.cpp

```diff
@@ -311,12 +332,28 @@ Value OptimizerUtils::boundValueWithMin(const meta::cpp2::ColumnDef& col, const
         case Value::Type::DATE : {
             return Value(Date());
         }
+        case Value::Type::TIME: {
+            return Value(Time());
+        }
         case Value::Type::DATETIME : {
             return Value(DateTime());
         }
-        default :
+        case Value::Type::__EMPTY__:
+        case Value::Type::NULLVALUE:
+        case Value::Type::VERTEX:
+        case Value::Type::EDGE:
+        case Value::Type::LIST:
+        case Value::Type::SET:
+        case Value::Type::MAP:
+        case Value::Type::DATASET:
+        case Value::Type::PATH: {
+            DLOG(FATAL) << "Not supported value type " << type
+                        << "for index.";
             return Value(NullType::BAD_TYPE);
+        }
     }
+    DLOG(FATAL) << "Unknown value type " << static_cast<int>(type);
```

Comment:
It's better not to add FATAL at here.

Reply:
It's exception that the type is never defined.